src/test/java/aquality/selenium/template/glue/stepdefinitions/DemoSteps.java

```diff
@@ -0,0 +1,61 @@
+package aquality.selenium.template.glue.stepdefinitions;
+
+import aquality.selenium.browser.AqualityServices;
+import aquality.selenium.template.configuration.Configuration;
+import aquality.selenium.template.forms.TopBarMenu;
+import aquality.selenium.template.forms.pages.ContactUsPage;
+import aquality.selenium.template.forms.pages.MainPage;
+import aquality.selenium.template.models.ContactUsInfo;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static aquality.selenium.template.forms.TopBarMenu.Item.CONTACT_US;
+
+public class DemoSteps {
+
+    private final MainPage mainPage = new MainPage();
+    private final TopBarMenu topBarMenu = new TopBarMenu();
+    private final ContactUsPage contactUsPage = new ContactUsPage();
+
+    @Given("Main page is opened")
+    public void mainPageIsOpened() {
+        AqualityServices.getBrowser().goTo(Configuration.getCurrentEnvironment().getStartUrl());
+    }
+
+    @When("I open Contact us page")
```

Comment:
Should the `Contact us` be a step parameter?

Reply:
I believe no. Not going to create one big step with switches;)

Reply:
you could show a way how to do it right :)